### ISSUE TITLE
[Feature Request] template name generator

### ISSUE THREAD

**Issue by hyzx86** [OrchardCMS/OrchardCore#13828]
The rules for template names are too complex. Can the Template name be generated through form configuration during template creation?

 https://docs.orchardcore.net/en/latest/docs/reference/modules/Templates/#content-part-templates

**Comment by hishamco** [comment]
I agree, if you have an idea please share it, at least we can do it as a community feature if not align with the official OC
